librz/util/ebcdic.c

```diff
@@ -547,7 +554,7 @@ RZ_API int rz_str_ibm290_to_ascii(const ut8 src, RZ_NONNULL RZ_OUT ut8 *dst) {
 RZ_API int rz_str_ibm290_from_ascii(RZ_NONNULL RZ_OUT ut8 *dst, const ut8 src) {
 	rz_return_val_if_fail(dst, 0);
 	*dst = ibm290_page00[src];
-	return 1;
+	return invalid_ebcdic_decode(src, *dst) ? 0 : 1;
```

Comment:
is this now boolean?

Reply:
No, it returns the number of bytes consumed. In case it couldn't decode a character, it returns `0`, because it consumed nothing. For UTF-8 it can return 0-4, for example.

I refactored all the other "code point" decoding functions like this as well. Because before it was kinda random. Should document it though.